src/main.cpp

```diff
@@ -1387,17 +1387,20 @@ unsigned int static GetNextWorkRequired_DIGI(const CBlockIndex* pindexLast, cons
     return bnNew.GetCompact();
 }
 
-
+#include "diff_delta.h"
 unsigned int static GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHeader *pblock)
 {
+	static int nDeltaSwitchover = fTestNet ? 100 : 213500;
```

Comment:
I like this, can we expand this to all diff algo's? `nDGW3Switchover`, `nDigiSwitchover`, `nKGWSwitchover`...